website/docs/timeline.md

```diff
@@ -3,40 +3,386 @@ title: Timeline
 toc: true
 ---
 
-## Timeline
-At its core, Hudi maintains a `timeline` of all actions performed on the table at different `instants` of time that helps provide instantaneous views of the table,
-while also efficiently supporting retrieval of data in the order of arrival. A Hudi instant consists of the following components
+A Hudi table maintains all operations happened to the table in a single timeline comprised of two parts, an active timeline and an archived timeline. The active timeline stores all the recent instants, while the archived timeline stores the older instants. An instant is a transaction where all respective partitions within a base path have been successfully updated by either a writer or a table service. Instants that get older in the active timeline are moved to archived timeline at various times.
 
-* `Instant action` : Type of action performed on the table
-* `Instant time` : Instant time is typically a timestamp (e.g: 20190117010349), which monotonically increases in the order of action's begin time.
-* `state` : current state of the instant
+An instant can alter one or many partitions:
 
-Hudi guarantees that the actions performed on the timeline are atomic & timeline consistent based on the instant time.
+-   If you have one batch ingestion, you’ll see that as one commit in the active timeline. If you ingest streaming data, you might see multiple commits in the active timeline.When you open that commit file, you’ll see a JSON object with metadata about how one or more partitions were altered.
```

Comment:
I don't follow this line. regardless of batch or streaming engines, 1 write is 1 commit. no?

Reply:
>you’ll see a JSON object with metadata about how one or more partitions were altered

we track file level changes actually. and it's AVRO,right